carma_planning_msgs/srv/GetEmergencyRoute.srv

```diff
@@ -0,0 +1,20 @@
+#
+# GetEmergencyRoute.srv
+# 
+# Provided by the EmergencyResponseVehicle node. Call the service to load a predefined 
+# route. Responds with success boolean and name of loaded route
+#
+# @author Alex Frye
+# @version 0.1
+#
+
+# Request - No arguments
+
+---
+# Response
+
+# Success boolean
+boolean is_successful
```

Comment:
I think `boolean` will need to be changed to `bool` since boolean is not a ROS msg primitive type.